src/server/adminInviteUser.ts

```diff
@@ -36,30 +47,31 @@ export async function adminInviteUser(
     });
 
     if (error) {
+        const logId = await logErrorReturnLogId(`Error with inviting user: ${error}`);
```

Comment:
How is this log displayed? I'm worried about this `error` being a class instance.

Reply:
Will it give us more info if we do this? I think we prefer sticking the error into the metadata object rather than trusting the string interpolation anyway.

```
logErrorReturnLogId(`Error with inviting user`, { error });
```